app.py

```diff
@@ -161,7 +168,7 @@ def get_current_frame_range(self, engine):
 
         return (current_in, current_out)
 
-    def set_frame_range(self, engine, in_frame, out_frame):
+    def set_frame_range(self, engine, in_frame, out_frame, prev_in, prev_out):
```

Comment:
`set_frame_range` is hopefully only used in this app.. or otherwise we would be abusing how the tank apps system work **if I'm not mistaken**   ,  @JeanChristopheMorin  ??